js/notify/settings/configure-notification-position.js

```diff
@@ -72,13 +72,27 @@ function getTemplatePath() {
  * @param windowName
  */
 function openConfigurationWindow(windowName) {
-    let allWindows = BrowserWindow.getAllWindows();
-    allWindows = allWindows.find((window) => { return window.winName === windowName });
+    const allWindows = BrowserWindow.getAllWindows();
+    const selectedParentWindow = allWindows.find((window) => { return window.winName === windowName });
 
     // if we couldn't find any window matching the window name
     // it will render as a new window
-    if (allWindows) {
-        windowConfig.parent = allWindows;
+    if (selectedParentWindow) {
+        windowConfig.parent = selectedParentWindow;
+
+        // This is a temporary work around until there
+        // is a fix for the modal window in windows from the electron
```

Comment:
this there an electron bug for this?  if so, put here.

Reply:
Electron issue [here](https://github.com/electron/electron/issues/10721)

Reply:
i would add that to documentation in code.